ApollosProject/apollos-apps
# fix: Prayer Request Post Headers

## ✏️ Description

The Avatar in the header of the Prayer Request Post got misaligned at some point and this PR resolves that issue and brings it into alignment with what's in Figma

## 🔬 To Test
1. Open up your Community Feed
2. Check out the Prayer Request Post Type

## 📸 Screenshots
| Before | After |
| --- | --- |
| ![CleanShot 2023-01-24 at 08 50 16](https://user-images.githubusercontent.com/45076058/214318473-4dfb28db-e0bd-4514-99b0-fd6d2f4fc08c.png) | ![CleanShot 2023-01-24 at 09 04 54](https://user-images.githubusercontent.com/45076058/214318505-e5caefb1-5c8a-40c2-978a-bf0c0fd7ca35.png) |

## ⚠️ To-do Before Merge
n/a

## 📝 Checklist
